pkg/apis/storage/types.go

```diff
@@ -672,3 +672,48 @@ type CSIStorageCapacityList struct {
 	// Items is the list of CSIStorageCapacity objects.
 	Items []CSIStorageCapacity
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeAttributesClass represents a class of volume attributes. It holds the mutable
+// attributes of volumes for the provisioner that should create and update volumes.
```

Comment:
VolumeAttributesClass represents a specification of mutable volume attributes defined by the storage driver. The class can be specified during dynamic provisioning of PersistentVolumeClaims, and changed after provisioning.